### PR TITLE
Sidebar fix

### PR DESCRIPTION
**Pull request by nkelava**
This PR adds fixes and adjustments for the sidebar.
It will:

 - fix toggle icon rotation that wasn't working
 - adjust toggle icon position
 - adjust collapse-expand transitions
 - adjust sidebar width
 - adjust container item position
 - add title attribute to sidebar links and toggle button for extra information while sidebar is collapsed or expanded

### CODE REVIEW

**Review by bdeak4 — APPROVED** [review]
LGTM 🚀

File: frontend/src/layouts/SideBar/SideBar.vue [hunk 1]
```diff
@@ -16,6 +16,7 @@ import { useUserStore } from '@/stores/user';
 const collapsed = useStorage('collapsed', false);
 const toggleMenu = () => (collapsed.value = !collapsed.value);
 const isCollapsed = computed(() => ({ expanded: !collapsed.value }));
+const buttonTitle = computed(() => (collapsed.value ? 'Expand menu' : 'Collapse menu'));
```
Comment (abasic) [inline frontend/src/layouts/SideBar/SideBar.vue:19]: Nitpick: no need for the brackets around the ternary operator.

**Review by abasic — APPROVED** [review]
Minor nitpick

File: frontend/src/layouts/SideBar/SideBar.vue (hunk 1 above)
Comment (nkelava) [reply frontend/src/layouts/SideBar/SideBar.vue:19]: Fully aware of that but ESLint is enforcing that rule. Will go with ESLint for now.

**Pull request merged**